syntax-markdown-flavor/syntax-markdown-flavor-github/src/main/java/org/xwiki/contrib/rendering/markdown/flavor/github/internal/parser/MarkdownGithubStreamParser.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.rendering.markdown.flavor.github.internal.parser;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.contrib.rendering.markdown.markdown12.internal.parser.Markdown12StreamParser;
+import org.xwiki.rendering.syntax.Syntax;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import static org.xwiki.contrib.rendering.markdown.flavor.github.internal.parser.MarkdownGithubParser.MARKDOWN_GITHUB;
+
+/**
+ * GitHub Flavored Markdown Stream Parser.
+ *
+ * @version $Id$
+ * @since 8.7
+ */
+@Component
+@Named("markdown+github/1.0")
+@Singleton
+public class MarkdownGithubStreamParser extends Markdown12StreamParser
```

Comment:
Those changes are not enough. Here you should also inject your own Configuration component for Github Markdown. Take a look on AbstractMarkdownStreamParser, you might need to modify it to allow injecting another configuration.